std/assembly/math.ts

```diff
@@ -2592,7 +2632,40 @@ export namespace NativeMathf {
       if (y == 1.0) return x;
       if (y == 0.0) return 1.0;
     }
-    return powf_lut(x, y);
+    if (ASC_SHRINK_LEVEL < 1) {
+      // see: musl/src/math/powf.c
+      return powf_lut(x, y);
+    } else {
+      // based on: metallic/src/math/float/powf.c
```

Comment:
The file linked here, [metallic/src/math/float/powf.c](https://github.com/jdh8/metallic/blob/master/src/math/float/powf.c), doesn't appear to resemble the following code. Am I looking at the wrong file or is this expected?

Reply:
It not exactly the same but it based on metallic version with manually inlined and slightly reworked part for handling special cases